packages/synapse-react-client/src/utils/functions/FileHandleUtils.test.ts

```diff
@@ -4,41 +4,55 @@ import {
   EXTERNAL_FILE_HANDLE_CONCRETE_TYPE_VALUE,
   EXTERNAL_OBJECT_STORE_FILE_HANDLE_CONCRETE_TYPE_VALUE,
   ExternalFileHandle,
+  ExternalGoogleCloudUploadDestination,
   ExternalObjectStoreFileHandle,
+  ExternalObjectStoreUploadDestination,
+  ExternalS3UploadDestination,
+  ExternalUploadDestination,
+  FileEntity,
   GOOGLE_CLOUD_FILE_HANDLE_CONCRETE_TYPE_VALUE,
   GoogleCloudFileHandle,
   PROXY_FILE_HANDLE_CONCRETE_TYPE_VALUE,
   ProxyFileHandle,
   S3_FILE_HANDLE_CONCRETE_TYPE_VALUE,
   S3FileHandle,
-} from '@sage-bionetworks/synapse-types'
-import {
-  ExternalGoogleCloudUploadDestination,
-  ExternalObjectStoreUploadDestination,
-  ExternalS3UploadDestination,
-  ExternalUploadDestination,
   S3UploadDestination,
   UploadType,
 } from '@sage-bionetworks/synapse-types'
-import { MOCK_USER_ID } from '../../../src/mocks/user/mock_user_profile'
+import mockFileEntityData from '../../mocks/entity/mockFileEntity'
+import {
+  mockExternalObjectStoreFileHandle,
+  mockFileHandle,
+} from '../../mocks/mock_file_handle'
+import { MOCK_USER_ID } from '../../mocks/user/mock_user_profile'
 import {
   getDataFileHandle,
   getFileHandleStorageInfo,
   getStorageLocationName,
   getUploadDestinationString,
-} from '../../../src/utils/functions/FileHandleUtils'
+} from './FileHandleUtils'
+import { SYNAPSE_STORAGE_LOCATION_ID } from '../../synapse-client'
+import {
+  mockExternalGoogleCloudUploadDestination,
+  mockExternalObjectStoreUploadDestination,
+  mockExternalS3UploadDestination,
+  mockExternalUploadDestination,
+  mockS3UploadDestination,
+} from '../../mocks/mock_upload_destination'
 
 describe('File Handle Utils', () => {
   describe('getDataFileHandle', () => {
     it('Returns the data file handle ID if it exists', () => {
       const dataFileHandleId = '123'
       const fileHandle = {
+        ...mockFileHandle,
```

Comment:
Changes in this describe block are related to lint errors raised after colocating test with component